app/controllers/logs_controller.rb

```diff
@@ -271,19 +271,23 @@ def update
   # can be given a single id or a list of ids
   def take
     unless params[:ids].present?
```

Comment:
Yeah that makes sense. I missed that fix. Feel free to make a quick pr for that!